### ISSUE TITLE
Feature Request -: bot can generate it's selfie photo when asked 

### ISSUE THREAD

**Issue by TruthSearchers** [chrisrude/oobabot#21]
Hey there is it possible to add a argument
--image (image given locally)
Like when we say generate a selfie or pic of yourself 
It generate it's multiple variant of image given locally through argument based on prompt 
on chat (selfie) plus also generate other images  (i mean based on prompt just like now) 
Both combined

just like there is extension sd_api injection type in ooobaboga 

It will be amazing

**Comment by Skrownerve** [comment]
I like this idea. I don't quite understand all the specifics being asked for, but here's my idea: we can specify a base image that the bot considers its own selfie. Then, it can run img2img on that image with various parameters either built-in or in a user request, and give the result.

Alternatively, we could specify a folder full of images meant to represent the bot, and then when asked for a selfie it picks one of them and sends it. OR it could then do img2img on it. Or both, both could be options. 😄

**Comment by TruthSearchers** [comment]
Yeah img2img with a prompt let's suppose user asked for i want selfie with a car so it generate that character image with a car 

Plus if user asked generate an image of Elon Musk (text to image ) , it generate image (just like now )

**Comment by chrisrude** [comment]
Interesting.  Something to keep in mind.  What do folks think of using the bot's discord avatar as an image source?  That way this feature could potentially generate images for other users as well.

**Comment by TruthSearchers** [comment]
If it's possible it will be cool

**Comment by TheMeIonGod** [comment]
> Interesting. Something to keep in mind. What do folks think of using the bot's discord avatar as an image source? That way this feature could potentially generate images for other users as well.

You could also have it "see" itself as a character that is trained in Lora (or something similar). For example, if you asked it to make an image of itself, it would put <lora:ainoku_HatsuneMiku:0.4> first, and then it would describe the selfie. I say this because img2img can be hit and miss especially with something like this.
